chapters/dashboard/log-stream.md

```diff
@@ -0,0 +1,145 @@
+## Start the Log Server
```

Comment:
For this page, can you try using this template? https://github.com/jina-ai/docs/pull/21 

I know your approach is simpler, but it's an attempt to have a standard look throughout the docs.